instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/javaagent/build.gradle.kts

```diff
@@ -26,11 +26,6 @@ testing {
     // Very different codepaths when threadlocals are enabled or not so we check both.
     // Regression test for https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/2403
     val testDisableThreadLocals by registering(JvmTestSuite::class) {
-      sources {
-        java {
-          setSrcDirs(listOf("src/test/java"))
-        }
-      }
```

Comment:
You shouldn't have removed this. The intent here is that this test suite uses the same test sources as the main test. If you remove it then it will use `src/testDisableThreadLocals/java` as test sources, as that directory does not exist it won't run any tests.

Reply:
Thank you for your answer